lib/ably/models/token_request.rb

```diff
@@ -14,10 +15,7 @@ def self.TokenRequest(attributes)
   end
 
 
-  # TokenRequest is a class that stores the attributes of a token request
-  #
-  # Ruby {http://ruby-doc.org/core/Time.html Time} objects are supported in place of Ably ms since epoch time fields.  However, if a numeric is provided
```

Comment:
> Ruby Time objects are supported in place of Ably ms since epoch time fields

I think this means we will need to change a lot of these comments :/

Reply:
I have reverted back all similar comments.